packages/core/useFetch/index.ts

```diff
@@ -213,21 +211,15 @@ export function useFetch<T>(url: MaybeRef<string>, ...args: any[]): UseFetchRetu
   }
 
   const setMethod = (method: string) => (payload?: unknown, payloadType?: PayloadType) => {
-    if (initialized) {
-      config.method = method
-      config.payload = payload
-      config.payloadType = payloadType || typeof payload === 'string' ? 'text' : 'json'
-      return shell as any
-    }
-    return undefined
+    config.method = method
+    config.payload = payload
+    config.payloadType = payloadType || typeof payload === 'string' ? 'text' : 'json'
+    return shell as any
   }
 
   const setType = (type: DataType) => () => {
-    if (initialized) {
```

Comment:
Since initialized was not called until the execute function was called I opted to remove this which fixed a bug when calling `.json()` or any of the other methods. Is there a particular reason to have this or is it okay to remove?

Reply:
Ah, it's `!initialized` 😅

Reply:
Haha righto just fixed it